triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh

```diff
@@ -108,9 +104,9 @@ if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
 fi
 
 if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
-    cp $pretrained_model_dir/exp/encoder_fp16.onnx $model_repo_path/encoder/1/encoder.onnx
-    cp $pretrained_model_dir/exp/decoder_fp16.onnx $model_repo_path/decoder/1/decoder.onnx
-    cp $pretrained_model_dir/exp/joiner_fp16.onnx $model_repo_path/joiner/1/joiner.onnx
+    cp $pretrained_model_dir/exp/encoder-epoch-9999-avg-1.onnx $model_repo_path/encoder/1/encoder.onnx
```

Comment:
Also, could we try use polygraphy or onnxmltools code to generate  fp16 onnx files? Then we could make the onnx recipe using fp16.

Actually, I have not tried use  $pretrained_model_dir/exp/encoder-epoch-9999-avg-1.int8.onnx. If it works with triton, we might could choose one solution between int8 and fp16 based on throughput.

Reply:
Currently, the native icefall/export-onnx.py will not generate the FP16 model so I keep FP32 here. 
But I can test with int8 to see whether these models can be used in Triton. Will keep you updated

Reply:
Thanks. Btw, "polygraphy convert --fp-to-fp16 -o encoder_fp16.onnx encoder-epoch-9999-avg-1.onnx" will be helpful.